debezium-connector-postgres/src/main/java/io/debezium/connector/postgresql/PostgresConnectorConfig.java

```diff
@@ -473,6 +473,8 @@ public static SchemaRefreshMode parse(String value) {
     public static final Field PORT = RelationalDatabaseConnectorConfig.PORT
             .withDefault(DEFAULT_PORT);
 
+    public static final String DEFAULT_UNAVAILABLE_NUMBER_ARRAY_PLACEHOLDER = "28694827,62948267,94852637,62847362,94837264,73486259,29483726,43827619,84692735,29873462";
```

Comment:
What does this number sequence mean?

Reply:
There is no specific meaning. It's random sequence of numbers. Happy to replace it with something else.

Reply:
@prburgu ok, thinking about that, do we really need a separate config option then? Can't we just call `getUnavailableValuePlaceholder()` returning `byte[]` and convert it into appropriate array type?

Reply:
Yeah, we can do that. Do we always convert 4 bytes into one integer, or do we convert to integer depending on the input type (I.e. int vs bigint)?

Reply:
@prburgu I think it would be unnecessary overkill. I added a commit that just converts byte[] to int[] or long[].